Sources/RInAppMessaging/Utilities/CommonUtility.swift

```diff
@@ -102,6 +102,15 @@ internal struct CommonUtility {
         Logger.debug("Failed converting value \(attribute.value) to \(attribute.type)")
         return nil
     }
+    
+    static func isValidURL(_ urlString: String) -> Bool {
```

Comment:
1. Can't URL constructor be used to validate this? why regex?
2. Does this work with deep link string?